### PR TITLE
Added composer script to run both PHPUnit test suites

### LINKED ISSUE #173: Consider adding a composer script for running both PHPUnit test suites
We have a composer script that runs PHPCS for the framework and the tests.
It would be great to add a similar command for the PHPUnit test suites.

### PR DESCRIPTION
**Pull request by christophherr**
Closes #173

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -66,6 +66,10 @@
       "@phpcs-tests"
     ],
     "test-unit": "\"vendor/bin/phpunit\" --testsuite unit --color=always",
-    "test-integration": "\"vendor/bin/phpunit\" --testsuite integration --configuration tests/phpunit/integration/phpunit.xml.dist --color=always"
+    "test-integration": "\"vendor/bin/phpunit\" --testsuite integration --configuration tests/phpunit/integration/phpunit.xml.dist --color=always",
+    "test": [
```
Comment (hellofromtonya) [inline composer.json]: Hmm, great idea.  But the word "test" might have a future task assigned to it from a package or composer.  Instead, how about if we give it a unique task name?

Maybe something like:

```
composer run-tests
```

Therefore, the script then would be `"run-tests"`.

Comment (christophherr) [reply composer.json]: Good point!
Should we brand it as 
`composer beans-tests` ?

Comment (hellofromtonya) [reply composer.json]: We could.  BUT it becomes less reusable at that point.  I like defining scripts that I can train my hands to run over and over again regardless of what package I'm in.

Let's go with `"run-tests"`.  Shoot, we can do the grouped sniffers too by saying: `composer run-phpcs`.

Comment (christophherr) [reply composer.json]: Yes, keeping it consistent would be great.
Should I add the change to the PHPCS script or do you want a separate PR for that?

Comment (hellofromtonya) [reply composer.json]: Separate PR would be good since this one is specific for tests.

**Pull request merged**